src/seedu/addressbook/data/person/Phone.java

```diff
@@ -40,6 +40,11 @@ public static boolean isValidPhone(String test) {
     public String toString() {
         return value;
     }
+    
+    @Override
+    public String getPrintableString(Printable... value) {
```

Comment:
You were expected to implement a getPrintableString() method for each of the classes. Your implementation here is not correct as we discussed in the class.